UI/question.html

```diff
@@ -0,0 +1,92 @@
+                    </li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <section id="main">
+        <div class="container">
+            <h1 class="page-title">Top Questions</h1>
+            <div class="question">
+                <h3>How do I become the best programmer in the universe?</h3>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+            </div>
+            <hr>
+            <h1>3 Answers</h1>
+            <div class="answers">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras faucibus quis sapien eu tempor. Suspendisse
+                    vestibulum felis vel ornare porttitor. Duis at tellus facilisis, mollis mi sit amet, porta quam. Integer
+                    sit amet pellentesque quam, non volutpat lectus. Vivamus hendrerit justo sed augue viverra, elementum
+                    maximus tortor vehicula. Donec vitae sem odio. Donec sed eros eu ipsum pulvinar congue. Sed arcu sapien,
+                    mollis quis sodales et, pharetra at est.</p>
+                <hr>
+            </div>
+            <div class="answer">
+                <h1>Your Answer</h1>
+                <form action="">
+                    <label for="uname">
+                        <b>Title</b>
```

Comment:
Should an answer have a title ? if so what would the title be in this case?

Reply:
Thanks for the reviews @krmroland ..